README.md

```diff
@@ -1,3 +1,7 @@
+<p align="center">
+  <img src="https://github.com/sequelize/sequelize/blob/main/docs/images/logo.png" />
```

Comment:
Here the same, so please make this an internal reference to the file so when you go back to another tag or commit you use the logo from that commit instead of the main

Reply:
Changed to internal link